implementation/configuration/src/configuration_impl.cpp

```diff
@@ -173,6 +174,8 @@ configuration_impl::configuration_impl(const configuration_impl &_other)
     sd_port_ = _other.sd_port_;
     sd_protocol_ = _other.sd_protocol_;
 
+    is_suppress_events_enabled_ = _other.is_suppress_events_enabled_;
+
```

Comment:
are this changes also part of the fix? as they were not mentioned on Aram's comment https://github.com/COVESA/vsomeip/issues/730

Reply:
They were discussed in the vsomeip meeting at the same time he was discussing the other issues.